docs/pages/x/api/tree-view/tree-view.json

```diff
@@ -41,9 +41,7 @@
         "describedArgs": ["event", "nodeIds"]
       }
     },
-    "selected": {
-      "type": { "name": "union", "description": "Array&lt;string&gt;<br>&#124;&nbsp;string" }
-    },
+    "selected": { "type": { "name": "any" } },
```

Comment:
I can try to improve the AST analysis, but the Autocomplete has the same limitation